tests/gql_behave/continuous_integration

```diff
@@ -104,7 +104,7 @@ class MemgraphRunner:
         memgraph_binary = os.path.join(self.build_directory, "memgraph")
         args_mg = [
             memgraph_binary,
-            "--storage-properties-on-edges",
+            "--storage-properties-on-edges=true",
```

Comment:
When the flag wasn’t set, the GQLBehave tests would fail with messages that this flag was turned off (and the default config value seems to be false: [link](https://github.com/memgraph/memgraph/blob/master/src/flags/general.cpp#L72)). I’ve encountered this both locally and on here.